apps/web/playwright/locale.e2e.ts

```diff
@@ -282,7 +282,7 @@ test.describe("authorized user sees correct translations (pt-br)", async () => {
   test("should return correct translations and html attributes", async ({ page, users }) => {
     await test.step("should create a pt-br user", async () => {
       const user = await users.create({
-        locale: "pt-br",
+        locale: "pt-BR",
```

Comment:
This works. next-i18next.config uses `"pt-BR"`

Reply:
Oh wow...